test/amd-test.js

```diff
@@ -0,0 +1,25 @@
+(function(define) {
+    'use strict';
+
+    define(function() {
+        return {
+            test: function() {
```

Comment:
We could have a main file that's written as a UMD module, then import it into a CommonJS test file and a AMD test file? it's a little overcomplex for something we're not actually going to run, just lint, but it would cut down on duplicated code... I'll throw something together and see how it looks and if there's too much boilerplate.